### ISSUE TITLE
Videos are not loading

### ISSUE THREAD

**Issue by 1stKaiser** [MrKrabat/plugin.video.crunchyroll#10]
As of 9/27, none of the videos work for me while using Kodi. Prior to that it worked fine for months.

**Comment by MrKrabat** [comment]
Can you post your Kodi log?
For me it is working without any issues

You can try to delete all files in `Kodi\userdata\addon_data\plugin.video.crunchyroll` and then reconfigure the addon

**Comment by 1stKaiser** [comment]
[kodi-log-cr-mrkrabat.txt](https://github.com/MrKrabat/plugin.video.crunchyroll/files/2430655/kodi-log-cr-mrkrabat.txt)

I did as you instructed and unfortunately still incur the same problem. Here is a log of what's going on while troubleshooting. I'm running Kodi 17.6 on Windows 10. Was working perfect until 2 days ago. Hope this helps. Thanks!

(P.S. I've been a member of VRV for as long as I had this plugin and it worked fine. Idk if you're a member of it and if it may have something to do with the problem. I wonder if it is DRM-related?)

**Comment by MrKrabat** [comment]
Its not a DRM problem, you are getting a "403 Forbidden" error from the server. This can have many causes.

Crunchyroll has a windows 10 app, is this app still able to play videos?

**Comment by 1stKaiser** [comment]
Yes, using the Windows 10 app, I can still play CR videos just fine. They play fine on the website directly, and they play fine on VRV. Its just using Kodi that it stopped working recently.

**Comment by 1stKaiser** [comment]
Alright. After a lot of fiddling with settings, I seemed to have found the problem. A lot of anime I was choosing was not working, but then I found an anime that was working lol. So I decided to check settings.

For this addon, I've always had the Video Quality setting at 1080p. I decided to set it to Adaptive and everything works now. Not sure why it recently stopped working. But it looks like the way CR videos are accessed might be conflicting with that setting. Thanks for working with me & I'll let you know if anything else happens.

And lastly, thanks for keeping this addon alive. :)

**Comment by robofunk** [comment]
Adaptive is not outputting 1080p anymore which is unfortunate.

**Comment by MrKrabat** [comment]
Adaptive is outputting 1080p.

You have to open the inputstream.adaptive settings and change it, it defaults to maximum 720p.

**Comment by 1stKaiser** [comment]
Yes, I made the change in the settings as well & set the max to 1080p, that way it should play the highest quality video if it is available. Otherwise everything still works as intended now. :)

**Comment by robofunk** [comment]
Seems to be the specific new episode that's missing the 1080p stream hopefully Cruncyroll will sort it out.
